### DISCUSSION TITLE
Welcome to hfsexplorer Discussions!

### DISCUSSION THREAD

**Discussion by unsound** [unsound/hfsexplorer#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NoviceCPU** [comment]
Hello, I'm very novice when it comes to technology but have always had an interest. I would love to gain some insight on hfsexplorer @

**Comment by Darumono** [comment]
thank you so much for this program, i couldn't reupload my photos to my pc TT
